decred/decred/dcr/dcrdata.py

```diff
@@ -624,7 +528,7 @@ def txsForAddr(self, addr):
         Args:
             addrs (string): Base-58 encoded address
         """
-        addrInfo = self.dcrdata.insight.api.addr.txs(addr)
+        addrInfo = self.dcrdata.insight.api.addr(addr)
```

Comment:
The `txs` path resolves to `/api/insight/api/addr` anyway. `txs` does not appear to be a valid `{command}` for that endpoint. See [here](https://github.com/decred/dcrdata/blob/84eff3bb0342ea6af775eabec0a4711f9a25edc6/api/insight/apirouter.go#L92-L98) and [here](https://github.com/decred/dcrdata/blob/84eff3bb0342ea6af775eabec0a4711f9a25edc6/api/insight/apiroutes.go#L1156).